users/models.py

```diff
@@ -1,3 +1,39 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import RegexValidator
 
-# Create your models here.
+
+# Creating a Skill model for allowing a user to have multiple unique skills
+class Skill(models.Model):
+    skill = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return self.skill
+
+
+# Creating a custom path for storing the user photos
+# Example : /MEDIA_ROOT/photos/1234567890/abc.jpg
+def path(instance, filename):
+    return 'photos/{0}/{1}'.format(instance.mobile, filename)
+
+
+# Creating a custom User model
+class User(AbstractUser):
```

Comment:
Change the name to something other than User, it might conflict later with Django User

Reply:
It shouldn't conflict since we have pointed settings.AUTH_USER_MODEL to this custom user, but I'll change it nevertheless.